request/request_json.go

```diff
@@ -0,0 +1,5 @@
+package request
+
+func (r *Request) AsJson() ([]byte, error) {
```

Comment:
exported method Request.AsJson should have comment or be unexported<br>func AsJson should be AsJSON